specification/redis/Cache.Management/RedisCacheAccessPolicy.tsp

```diff
@@ -0,0 +1,68 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Cache;
+/**
+ * Response to get/put access policy.
+ */
+@parentResource(RedisResource)
+model RedisCacheAccessPolicy
+  is Azure.ResourceManager.ProxyResource<RedisCacheAccessPolicyProperties> {
+  ...ResourceNameParameter<
+    Resource = RedisCacheAccessPolicy,
+    KeyName = "accessPolicyName",
+    SegmentName = "accessPolicies",
+    NamePattern = "^([a-zA-Z0-9][a-zA-Z0-9- ]*[a-zA-Z0-9]|[a-zA-Z0-9])$"
+  >;
+}
+
+@armResourceOperations
+interface RedisCacheAccessPolicies {
+  /**
+   * Gets the detailed information about an access policy of a redis cache
+   */
+  get is ArmResourceRead<RedisCacheAccessPolicy>;
+
+  /**
+   * Adds an access policy to the redis cache
+   */
+  createUpdate is ArmResourceCreateOrReplaceAsync<
+    RedisCacheAccessPolicy,
+    LroHeaders = ArmLroLocationHeader & Azure.Core.Foundations.RetryAfterHeader
+  >;
+
+  /**
+   * Deletes the access policy from a redis cache
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-delete-operation-response-codes" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  delete is ArmResourceDeleteWithoutOkAsync<
```

Comment:
Does this get grandfathered in as backcompat, or do you leave the fixme for us to consider fixing in future versions?